src/context/TreeContext.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from "react";
+
+interface TreeNode {
+  name: string;
+  props: Array<PropFormValue>;
+  state: Array<StateFormValue>;
+  parent: string;
+  children?: Array<TreeNode>;
```

Comment:
`TreeNode[]`

Reply:
The code has changed on those lines, but I still modified with what you suggested on the spots where TreeNode[] is used.